Bio/ExPASy/Enzyme.py

```diff
@@ -55,7 +55,7 @@ def read(handle):
 
 
 class Record(dict):
-    """\
+    r"""\
```

Comment:
On reading the whole docstring, I suspect that trailing slash is the only reason ``pydocstyle`` complained. i.e. Does this work with no raw string mode:

```
class Record(dict):
    """Holds from an ExPASy ENZYME record as a Python dictionary.

    Each record contains the following keys:
    ...
```